src/main/java/com/github/splendor_mobile_game/websocket/handlers/exceptions/CardDoesntExistException.java

```diff
@@ -0,0 +1,18 @@
+package com.github.splendor_mobile_game.websocket.handlers.exceptions;
+
+public class CardDoesntExistException extends Exception{
```

Comment:
Exceptions should have short one line java doc stating the same what the class name at least XD I forgot about that in the previous pr.